src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -101,35 +119,59 @@ export const ColorInputPanel = React.forwardRef(
       }
 
       if (currentFormat === 'hsl') {
-        const [h, s, l] = input.map(Number);
+        const [h, s, l, a] = input.map(Number);
 
-        if (h < 0 || h > 360 || s < 0 || s > 100 || l < 0 || l > 100) {
+        if (
+          h < 0 ||
+          h > 360 ||
+          s < 0 ||
+          s > 100 ||
+          l < 0 ||
+          l > 100 ||
+          a < 0 ||
+          a > 1
+        ) {
           return;
         }
         const hsl = activeColor.toHslColor();
-        if (hsl.h === h && hsl.s === s && hsl.l === l) {
+        if (hsl.h === h && hsl.s === s && hsl.l === l && hsl.a === a) {
           return;
         }
 
-        color = ColorValue.create({ h, s, l });
+        color = ColorValue.create({ h, s, l, a });
+        alpha = a;
       }
 
       if (currentFormat === 'rgb') {
-        const [r, g, b] = input.map(Number);
+        const [r, g, b, a] = input.map(Number);
 
-        if (r < 0 || r > 255 || g < 0 || g > 255 || b < 0 || b > 255) {
+        if (
+          r < 0 ||
+          r > 255 ||
+          g < 0 ||
+          g > 255 ||
+          b < 0 ||
+          b > 255 ||
+          a < 0 ||
+          a > 1
+        ) {
           return;
         }
         const rgb = activeColor.toRgbColor();
-        if (rgb.r === r && rgb.g === g && rgb.b === b) {
+        if (rgb.r === r && rgb.g === g && rgb.b === b && rgb.a === a) {
           return;
         }
 
-        color = ColorValue.create({ r, g, b });
+        color = ColorValue.create({ r, g, b, a });
+        alpha = a;
       }
 
       if (color) {
-        applyHsvColorChange(color.toHsvColor(), true);
+        const hsv = color.toHsvColor();
+        if (showAlpha) {
+          hsv.a = alpha;
```

Comment:
The color value is already being created with `a`. Why do you need to set it again?

Reply:
Removed